2019-06-01T11:28:42.705+0000 I CONTROL  [main] Automatically disabling TLS 1.0, to force-enable TLS 1.0 specify --sslDisabledProtocols 'none'
2019-06-01T11:28:42.710+0000 I CONTROL  [initandlisten] MongoDB starting : pid=1 port=27017 dbpath=/data/db 64-bit host=3bba26432216
2019-06-01T11:28:42.711+0000 I CONTROL  [initandlisten] db version v4.0.10
2019-06-01T11:28:42.711+0000 I CONTROL  [initandlisten] git version: c389e7f69f637f7a1ac3cc9fae843b635f20b766
2019-06-01T11:28:42.712+0000 I CONTROL  [initandlisten] OpenSSL version: OpenSSL 1.0.2g  1 Mar 2016
2019-06-01T11:28:42.712+0000 I CONTROL  [initandlisten] allocator: tcmalloc
2019-06-01T11:28:42.712+0000 I CONTROL  [initandlisten] modules: none
2019-06-01T11:28:42.713+0000 I CONTROL  [initandlisten] build environment:
2019-06-01T11:28:42.713+0000 I CONTROL  [initandlisten]     distmod: ubuntu1604
2019-06-01T11:28:42.713+0000 I CONTROL  [initandlisten]     distarch: x86_64
2019-06-01T11:28:42.714+0000 I CONTROL  [initandlisten]     target_arch: x86_64
2019-06-01T11:28:42.714+0000 I CONTROL  [initandlisten] options: { net: { bindIpAll: true }, storage: { mmapv1: { smallFiles: true } }, systemLog: { destination: "file", path: "/var/log/mongodb/mongod.log" } }
2019-06-01T11:28:42.716+0000 W STORAGE  [initandlisten] Detected unclean shutdown - /data/db/mongod.lock is not empty.
2019-06-01T11:28:42.720+0000 I STORAGE  [initandlisten] Detected data files in /data/db created by the 'wiredTiger' storage engine, so setting the active storage engine to 'wiredTiger'.
2019-06-01T11:28:42.724+0000 W STORAGE  [initandlisten] Recovering data from the last clean checkpoint.
2019-06-01T11:28:42.725+0000 I STORAGE  [initandlisten] wiredtiger_open config: create,cache_size=487M,session_max=20000,eviction=(threads_min=4,threads_max=4),config_base=false,statistics=(fast),log=(enabled=true,archive=true,path=journal,compressor=snappy),file_manager=(close_idle_time=100000),statistics_log=(wait=0),verbose=(recovery_progress),
2019-06-01T11:28:54.376+0000 I STORAGE  [initandlisten] WiredTiger message [1559388534:376113][1:0x7f2e7bb23a80], txn-recover: Main recovery loop: starting at 11/353024 to 12/256
2019-06-01T11:28:54.404+0000 I STORAGE  [initandlisten] WiredTiger message [1559388534:404612][1:0x7f2e7bb23a80], txn-recover: Recovering log 11 through 12
2019-06-01T11:28:54.855+0000 I STORAGE  [initandlisten] WiredTiger message [1559388534:855703][1:0x7f2e7bb23a80], txn-recover: Recovering log 12 through 12
2019-06-01T11:28:54.896+0000 I STORAGE  [initandlisten] WiredTiger message [1559388534:896298][1:0x7f2e7bb23a80], txn-recover: Set global recovery timestamp: 0
2019-06-01T11:28:54.917+0000 I RECOVERY [initandlisten] WiredTiger recoveryTimestamp. Ts: Timestamp(0, 0)
2019-06-01T11:28:54.980+0000 W STORAGE  [initandlisten] Detected configuration for non-active storage engine mmapv1 when current storage engine is wiredTiger
2019-06-01T11:28:54.981+0000 I CONTROL  [initandlisten] 
2019-06-01T11:28:54.981+0000 I CONTROL  [initandlisten] ** WARNING: Access control is not enabled for the database.
2019-06-01T11:28:54.982+0000 I CONTROL  [initandlisten] **          Read and write access to data and configuration is unrestricted.
2019-06-01T11:28:54.982+0000 I CONTROL  [initandlisten] 
2019-06-01T11:28:55.025+0000 I FTDC     [initandlisten] Initializing full-time diagnostic data capture with directory '/data/db/diagnostic.data'
2019-06-01T11:28:55.028+0000 I NETWORK  [initandlisten] waiting for connections on port 27017
2019-06-01T11:28:55.225+0000 I NETWORK  [listener] connection accepted from 172.29.0.1:54058 #1 (1 connection now open)
2019-06-01T11:28:55.227+0000 I NETWORK  [conn1] received client metadata from 172.29.0.1:54058 conn1: { driver: { name: "mongo-java-driver", version: "3T_5.1.1-478-gfd17fbf" }, os: { type: "Darwin", name: "Mac OS X", architecture: "x86_64", version: "10.13.6" }, platform: "Java/Oracle Corporation/1.8.0_131-b11" }
2019-06-01T11:28:55.294+0000 I NETWORK  [listener] connection accepted from 172.29.0.4:52774 #2 (2 connections now open)
2019-06-01T11:28:55.295+0000 I NETWORK  [conn2] received client metadata from 172.29.0.4:52774 conn2: { driver: { name: "PyMongo", version: "3.8.0" }, os: { type: "Linux", name: "Linux", architecture: "x86_64", version: "4.9.125-linuxkit" }, platform: "CPython 3.6.5.final.0" }
2019-06-01T11:28:55.300+0000 I NETWORK  [listener] connection accepted from 172.29.0.4:52776 #3 (3 connections now open)
2019-06-01T11:28:55.302+0000 I NETWORK  [conn3] received client metadata from 172.29.0.4:52776 conn3: { driver: { name: "PyMongo", version: "3.8.0" }, os: { type: "Linux", name: "Linux", architecture: "x86_64", version: "4.9.125-linuxkit" }, platform: "CPython 3.6.5.final.0" }
2019-06-01T11:28:55.304+0000 I COMMAND  [conn3] CMD: drop mall.timeline_goods
2019-06-01T11:28:55.306+0000 I STORAGE  [conn3] Finishing collection drop for mall.timeline_goods (0561a7cc-5e35-4ede-aa0a-1a85c4c6d708).
2019-06-01T11:28:55.455+0000 I STORAGE  [conn3] createCollection: mall.timeline_goods with generated UUID: 43bd27bc-7068-4439-bf73-c66d4a74d203
2019-06-01T11:28:55.823+0000 I NETWORK  [conn3] end connection 172.29.0.4:52776 (2 connections now open)
2019-06-01T11:28:55.823+0000 I NETWORK  [conn2] end connection 172.29.0.4:52774 (1 connection now open)
2019-06-01T11:28:56.015+0000 I FTDC     [ftdc] Unclean full-time diagnostic data capture shutdown detected, found interim file, some metrics may have been lost. OK
